extensions/positron-python/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "python",
-    "displayName": "Positron Python",
-    "description": "Positron Python",
+    "displayName": "Python",
+    "description": "Python language support for Positron's language server, console, editors, and more.",
```

Comment:
The R extension says:

> R Language Pack for Positron

What would you think about that here, for consistency?